### PR TITLE
Some suggested updates to the README

### PR DESCRIPTION
**Pull request by adrianludwin**
Hi @yugui, I tried to write down some of the instructions that I thought might have helped me when I first set out to use Go with Bazel. Are any of these worth pulling into the main repo?

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -90,6 +94,71 @@ They currently do not support (in order of importance):
     )
     ```
 
+* To depend on external libraries, you have two options: vendoring or external
```
Comment (pmbethe09) [inline README.md]: This is a pretty long section for a README, but the information is useful.
I don't know what other projects do, but can a 'Vendoring.md' be created with this info and linked to?

Comment (adrianludwin) [inline README.md]: Sure, I'll do that now.

**Comment by yugui** [comment]
Jenkins, please test this just in case.

... I hope the Jenkins integration works fine now.

**Comment by yugui** [comment]
LGTM. 
@pmbethe09, do you have any other comments?

**Pull request merged**